dispatcher/dispatcher.go

```diff
@@ -348,20 +371,12 @@ func (d *IotxDispatcher) HandleTell(ctx context.Context, chainID uint32, peer pe
 	case iotexrpc.MessageType_BLOCK_REQUEST:
 		d.dispatchBlockSyncReq(ctx, chainID, peer, message)
 	case iotexrpc.MessageType_BLOCK:
-		d.dispatchBlockCommit(ctx, chainID, message)
+		d.dispatchBlock(ctx, chainID, message)
 	default:
 		log.L().Warn("Unexpected msgType handled by HandleTell.", zap.Any("msgType", msgType))
 	}
 }
 
-func (d *IotxDispatcher) enqueueEvent(event interface{}) {
-	if len(d.eventChan) == cap(d.eventChan) {
-		log.L().Warn("dispatcher event chan is full, drop an event.")
-		return
-	}
-	d.eventChan <- event
-}
-
 func (d *IotxDispatcher) updateEventAudit(t iotexrpc.MessageType) {
```

Comment:
should update this stat as well, can defer to the next pr

Reply:
addressed